### PR TITLE
Ds templates

### PR DESCRIPTION
**Pull request by Zahaby-magalix**
-neo4j: the graph database platform powering today's mission-critical enterprise applications, including artificial intelligence, fraud detection and recommendations.

-rstudio: is a free and open-source integrated development environment (IDE) for R, a programming language for statistical computing and graphics.

-spark-zeppelin: fast and general-purpose cluster computing system

### CODE REVIEW

File: stable/neo4j/template.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			"port": 7474
+		}, {
+			"is_public": true,
+			"target_port": "bolt",
+			"port": 7687
+		}]
+	}],
+	"volumes": [{
+		"name": "neo4j-data",
+		"type": "persistent",
```
Comment (gagliardetto) [inline stable/neo4j/template.json]: The volume has no `type` field. But it has a `persistent: false` (bool) field.

Comment (Zahaby-magalix) [reply stable/neo4j/template.json]: I am sorry I generated this part from mgx dev console. 
I will confirm it with Furqan as well. 

Will fix this.

Comment (Zahaby-magalix) [reply stable/neo4j/template.json]: has been changed

File: stable/neo4j/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: "neo4j"
+description: "Neo4j: The graph database platform powering today's mission-critical enterprise applications, including artificial intelligence, fraud detection and recommendations."
+categories:
+  - "data science"
```
Comment (gagliardetto) [inline stable/neo4j/meta.yaml]: Please refer to the categories listed here: https://raw.githubusercontent.com/MagalixTechnologies/templates/master/categories.yaml

File: stable/rstudio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: "rstudio"
+description: "RStudio is a free and open-source integrated development environment (IDE) for R, a programming language for statistical computing and graphics."
+categories:
+  - "data science"
```
Comment (gagliardetto) [inline stable/rstudio/meta.yaml]: Please refer to the categories listed here: https://raw.githubusercontent.com/MagalixTechnologies/templates/master/categories.yaml

File: stable/spark-zeppelin/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+name: "spark-zeppelin"
+description: "Fast and general-purpose cluster computing system."
+categories:
+  - "data science"
```
Comment (gagliardetto) [inline stable/spark-zeppelin/meta.yaml]: Please refer to the categories listed here: https://raw.githubusercontent.com/MagalixTechnologies/templates/master/categories.yaml

File: unstable/cloudera/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name: "cloudera"
+description: "Cloudera's open-source Apache Hadoop distribution, CDH (Cloudera Distribution Including Apache Hadoop), targets enterprise-class deployments of that technology."
+categories:
+  - "data science"
```
Comment (gagliardetto) [inline unstable/cloudera/meta.yaml:4]: Please refer to the categories listed here: https://raw.githubusercontent.com/MagalixTechnologies/templates/master/categories.yaml

**Review by gagliardetto — CHANGES_REQUESTED** [review]

File: unstable/cloudera/meta.yaml (hunk 5 above)
Comment (Zahaby-magalix) [reply unstable/cloudera/meta.yaml:4]: changed it to : 
Data Science

Please check

Comment (Zahaby-magalix) [reply unstable/cloudera/meta.yaml:4]: changed "Demo Apps" as well

File: stable/dokuwiki/meta.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 name: "dokuwiki"
 description: "DokuWiki is a standards-compliant, simple to use wiki optimized for creating documentation. It is targeted at developer teams, workgroups, and small companies. All data is stored in plain text files, so no database is required."
 categories:
-  - "demo"
+  - "Demo Apps"
```
Comment (gagliardetto) [inline stable/dokuwiki/meta.yaml]: Please use `category.name`, not `category.display_name`

**Review by gagliardetto — CHANGES_REQUESTED** [review]

File: stable/dokuwiki/meta.yaml (hunk 6 above)
Comment (Zahaby-magalix) [reply stable/dokuwiki/meta.yaml]: I will do it now

**Pull request merged**